Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG JAVA_VERSION=16
+ARG JVM_FLAVOR=hotspot
+
+FROM adoptopenjdk:${JAVA_VERSION}-jdk-${JVM_FLAVOR} AS builder
+WORKDIR /builddir
+
+COPY gradlew ./
+COPY gradle/ gradle/
```

Comment:
Can't we just `COPY . ./`?

Reply:
See: https://github.com/PaperMC/bibliothek/pull/43#discussion_r646137550